### ISSUE TITLE
E 4438891747

### ISSUE THREAD

**Issue by camsmommy113** [Masudbro94/python-hacked-mobile-phone-#3]
E

_Originally posted by @LuuCidd in https://github.com/Kaushikdhola/Anti-Theft-App/issues/2_

**Comment by amiegreenwood56** [comment]
Yeah somebody is calling my phone or been marrying my phone I'm not for sure trying to figure it out I
